Sources/Commands/CMakeLists.txt

```diff
@@ -43,12 +43,8 @@ target_link_libraries(Commands PUBLIC
   Workspace
   XCBuildSupport
   Xcodeproj)
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
-  if(Foundation_FOUND)
-    target_link_libraries(Commands PUBLIC
-      FoundationXML)
-  endif()
-endif()
+target_link_libraries(Commands PRIVATE
```

Comment:
public to private intentional? to be clear that change seems fine but making sure its intentional

Reply:
Yeah, I didn't hit any build failures with that change.  `PRIVATE` should be preferred - it prevents all users of this library from linking against `FoundationXML` explicitly.